doc/source/f2py/usage.rst

```diff
@@ -21,7 +22,7 @@ Command ``f2py``
 =================
 
 When used as a command line tool, ``f2py`` has three major modes,
-distinguished by the usage of ``-c`` and ``-h`` switches:
+distinguished by the usage of ``-c``, ``-m`` and ``-h`` switches:
```

Comment:
``-m`` does not define a mode. It just specifies the name of the generated module. Suggest undo.

Reply:
Btw, when `-m` is not used, then it is equivalent to usage `-m untitled`